### ISSUE TITLE
RF localisation causes singularity issues 

### ISSUE THREAD

**Issue by odunbar** [CliMA/CalibrateEmulateSample.jl#288]
## issue
During optimization of RF hyperparameters, one of the loss function entries was zero-d out (a hack fix to remove this item).
It does not outwardly effect the calibration. But this causes ensembles in the optimisation algorithm to have a 0-uncertainty row, which causes an issue with the localisation algorithms that often invert the variance/covariance.

## Solution

Remove the item from the loss function&data instead of simply zeroing it out.
